src/main/java/org/ladocuploader/app/utils/SubmissionUtilities.java

```diff
@@ -5,8 +5,60 @@
 import java.text.DecimalFormat;
 import java.util.*;
 
-public class SubmissionUtilities {
+import static formflow.library.inputs.FieldNameMarkers.DYNAMIC_FIELD_MARKER;
 
+public class SubmissionUtilities {
+    public static final String ENCRYPTED_SSNS_INPUT_NAME = "householdMemberEncryptedSSN";
+
+    public static final Map<String, String> EDUCATION_MAP = new HashMap<>();
+    public static final Map<String, String> MARITAL_STATUS_MAP = new HashMap<>();
+
+    public static final Map<String, String> RELATIONSHIP_MAP = new HashMap<>();
+
+    static {
+        EDUCATION_MAP.put("firstGrade", "1st grade");
```

Comment:
Are these PDF specific?  They might be better as a PDF Related Enum then.  But balancing that with time and life of this project makes me think that what you've done is just fine. :)   Maybe you could tuck a "PDF" into the name of these just so we can easily tell what they are used for?

Reply:
Oh good point - will do!